### PR TITLE
Fixes #14586 - Associate default templates to Orgs and Locs during seed

### PR DESCRIPTION
**Pull request by ares**

### CODE REVIEW

File: db/seeds.d/07-provisioning_templates.rb [hunk 1]
```diff
@@ -79,6 +80,13 @@
       :snippet  => false,
       :template => File.read(File.join("#{Rails.root}/app/views/unattended", input.delete(:source)))
     }.merge(input))
+
+    created.push(t.name)
```
Comment (iNecas) [inline db/seeds.d/07-provisioning_templates.rb]: why not just updating the template taxonomies here as opposed to searching for them by name later?

Comment (ares) [inline db/seeds.d/07-provisioning_templates.rb]: Not sure what you mean by "template taxonomies". The reason I'm doing it this way is to support reoccurring seed runs, if user disassociate the template we should not re-associate it so we only associate those that were newly created in this seed run

**Comment by ares** [comment]
ah, I got what you mean, refactored

**Comment by dLobatog** [comment]
Merged as 2fb0d8d129588a71dcb953b12fe84d2fd8154467, thanks @ares!

**Pull request closed without merging**